File: MATEMÁTICA/2489/main.js
function h(a, d, r) {
    let x = (90 - r) * Math.PI / 180;
    return a - Math.tan(x) * d;
}

let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').trim().split('\n');
for (let line of lines) {
    let [a, d, r] = line.split(' ').map(Number);
    console.log(h(a, d, r).toFixed(4));
}

File: MATEMÁTICA/2489/main.py
from sys import stdin
from math import pi, tan


def h(a, d, r):
    x = (90 - r) * pi / 180
    return a - tan(x) * d


for line in stdin:
    a, d, r = map(float, line.split())
    print(f'{h(a, d, r):.4f}')
